src/Authentication.Abstractions/AzureTenant.cs

```diff
@@ -34,9 +34,28 @@ public class AzureTenant : IAzureTenant
         /// </summary>
         public string Directory { get; set; }
 
+        /// <summary>
+        /// Whether tenant category is home
+        /// </summary>
+        public bool IsHome { 
+            get {
+                return string.IsNullOrEmpty(this.GetProperty(Property.TenantCategory)) 
+                    || 0 == string.Compare(this.GetProperty(Property.TenantCategory), "Home", true);
```

Comment:
Suggest to use string.Equals instead of Compare for comparing string equalities

Reply:
Done